corehq/apps/smsbillables/async_handlers.py

```diff
@@ -138,8 +138,14 @@ def _directed_fee(direction, backend_api_id, backend_instance_id):
             return fmt_dollar_amount(usage_fee.amount + usd_gateway_fee)
 
         rate_table = []
+
+        from corehq.apps.sms.test_backend import TestSMSBackend
+
         for backend_instance in backends:
             backend_type = get_backend_by_class_name(backend_instance.doc_type)
+            # not a better way than hardcoding
+            if backend_type == TestSMSBackend:
```

Comment:
It's generally not a good idea to use == when checking for this because then if we subclass TestSMSBackend it doesn't work anymore.  So we have to wrap the SMSBackend correctly as I alluded to in https://github.com/dimagi/commcare-hq/pull/5940#discussion_r27036399

I just added a commit on this branch which adds wrap_correctly() to SMSBackend, so now you can just:

    backend_instance = backend_instance.wrap_correctly()

and from there, use isinstance to check if it's a TestSMSBackend.  You can also remove the comment about hard-coding since we're not hard-coding any backend ids, we're just excluding backends of a certain type, which is fine.